tests/ctest/common/path_tests.cpp

```diff
@@ -113,6 +113,49 @@ TEST(Path, Combine)
 #endif
 }
 
+
+#if !defined(__APPLE__)
+TEST(Path, EscapeCommandLine)
+{
```

Comment:
These tests are currently asserting that the EscapeCmdLine function does nothing, they should check that the arguments are quoted properly.